src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import './Footer.scss';
+
+const backToTop = () => {
+  window.scrollTo({
+    top: 0,
+    behavior: 'smooth',
+  });
+};
+
+export const Footer: React.FC = () => (
+  <footer className="footer">
+    <div className="footer__container">
+      <div className="footer__logo">
+        {/* <a href="/" className="footer__logo__link">
+          <Logo />
+        </a> */}
+      </div>
+
+      <ul className="footer__nav">
+        <li className="footer__nav__item">
+          <a
+            className="footer__nav__link"
+            href="https://github.com/fe-sep22-FixMePlease/product-catalog-frontend"
+            target="_blank"
+            rel="noreferrer"
+          >
+            github
+          </a>
```

Comment:
Цю штуку можна зробти простіше. Можна створити массив footerNavs, що буде складатись з об'єктів типу: 
`footerNavLinkInfo = {
	link: string,
	title: string,
	id: number
}` і відрендерити цей массив.
```suggestion
 	{footerNavs.map(footerNav => (
 	       <li className="footer__nav__item" key = {footerNav.id}>
	          <a
	            className="footer__nav__link"
	            href={footerNav.link}
	            target="_blank"
	            rel="noreferrer"
	          >
                      {footerNav.title}
          	</a>
          ))}
```